org.civicrm.styleguide/styleguide.php

```diff
@@ -123,3 +123,41 @@ function styleguide_civicrm_angularModules(&$angularModules) {
 function styleguide_civicrm_alterSettingsFolders(&$metaDataFolders = NULL) {
   _styleguide_civix_civicrm_alterSettingsFolders($metaDataFolders);
 }
+
+/**
+ * Implements hook_civicrm_navigationMenu().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_navigationMenu
+ */
+function styleguide_civicrm_navigationMenu(&$menu) {
+  _styleguide_civix_insert_navigation_menu($menu, 'Support/Developer', array(
+    'label' => ts('Style Guide', array('domain' => 'org.civicrm.styleguide')),
+    'name' => 'developer_styleguide',
+    'permission' => 'access CiviCRM',
+    'operator' => 'OR',
+    'separator' => 0,
+  ));
+  foreach (Civi::service('styleguides')->getAll() as $name => $defn) {
+    _styleguide_civix_insert_navigation_menu($menu, 'Support/Developer/developer_styleguide', array(
+      'label' => $defn['label'],
+      'name' => 'developer_styleguide_' . $defn['name'],
+      'url' => 'civicrm/styleguide/' . $defn['name'],
+      'permission' => 'access CiviCRM',
+      'operator' => 'OR',
+      'separator' => 0,
+    ));
+  }
+}
+
+/**
+ * Implements hook_civicrm_container().
+ *
+ * @link http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_container
+ */
+function styleguide_civicrm_container(\Symfony\Component\DependencyInjection\ContainerBuilder $container) {
+  $container->setDefinition('styleguides', new \Symfony\Component\DependencyInjection\Definition(
```

Comment:
Wouldn't it be better to name the service as something like `style_guides`, `style-guides` or `styleGuides`? I'm in favor of the first option, as it seems to be the standard in Symfony and its bundles